### DISCUSSION TITLE
How do I load and continue Flux Lora training from an existing safetensors Lora file?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [bghira/SimpleTuner#754] [Q&A] [answered]
I want to continue my training using a safetensors Lora file I have from a previous training session on a server. I only have the safetensors file for the Lora - no backups or other saved files due to the deletion of the previous server.
Is there a way to load pretrained Lora weights from the Flux Lora and proceed with training using new parameters?

**Comment by aa956** [comment, marked as answer]
Have you tried the method mentioned in the docs?

https://github.com/bghira/SimpleTuner/blob/main/OPTIONS.md

```text
  --init_lora INIT_LORA
                        Specify an existing LoRA safetensors file to
                        initialize the LoRA and continue training or finetune
                        an existing LoRA.
```

If it's not working for flux LoRA specifically then it's probably worth a bug report?

**Comment by a-l-e-x-d-s-9** [comment]
@aa956 
Yes, I have used it. 
It has been added after this question has been asked.
I only tested regular Lora, not sure it's working with Lycoris.
